### ISSUE TITLE
Incentivizing practices with less-harmful side effects, and/or intangible benefits

### ISSUE THREAD

**Issue by gcassel** [valnet/valuenetwork#459]
Hopefully the thread title indicates what I'm perceiving from this link:   http://www.bbc.com/news/technology-34872563   -- That's a sort of cryptocurrency generated by walking.   I bring it up here in an exceedingly generic way.   I imagine that similar ideas have come up before, but I don't have time to mine for them.   

I reckon there are at least three ways to go with my train of thought.  One way is to add recommended protocols for 'desired-practice incentives' to one or more general forms of Value Equation.   Another option is to emphasize that communities should always develop their own value equations, using some sort of API.  Another option, of course, would be to limit value equation forms to processes which directly add value to a defined community.

**Comment by bhaugen** [comment]
@gcassel not sure what to do with this. My personal feeling is that it's premature to recommend 'desired-practice incentives'. Our preference (as the current software developers) is to support the value equations desired by the networks that are using the software, and then look at the results.

**Comment by gcassel** [comment]
Thanks @bhaugen ; that certainly makes sense.   I'll be quite interested in eventually supporting some way for communities to plug their own value equations into a system, or to at least customize the equations quite a bit-- if that ever becomes a sustainable objective.   I have great confidence in your developmental priorities.

**Comment by bhaugen** [comment]
Well, as of now, the only way you can plug your value equations into a system is to use this one. The whole nine yards, as they say.

Our next trick is https://valueflo.ws/ where we do think it will be possible to plug value equations into linked open data from more than one system. But a lot more work needs to be done.

**Comment by gcassel** [comment]
Yeah, Value Flows is super important to me, and I hope I'll have time to focus more on it in the future.
